runtime/src/sorted_storages.rs

```diff
@@ -215,11 +215,19 @@ pub mod tests {
                 storage_count: 0,
             }
         }
+
+        pub fn new_for_tests(storages: &[&'a SnapshotStorage], slots: &[Slot]) -> Self {
+            SortedStorages::new_with_slots(storages.iter().cloned().zip(slots.iter()), None, None)
```

Comment:
Do we need to handle `storages.len() != slots.len()`? If not, can we add an assert for this.